sklearn/feature_extraction/text.py

```diff
@@ -635,6 +645,11 @@ class CountVectorizer(BaseEstimator, VectorizerMixin):
         in the range [0.7, 1.0) to automatically detect and filter stop
         words based on intra corpus document frequency of terms.
 
+      .. deprecated:: 0.20
```

Comment:
I think this (and all the similar deprecation messages) need an extra indent by 2 spaces, since the rendered section for stop words does not look good https://20519-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.feature_extraction.text.CountVectorizer.html

Reply:
I am not able to produce the documentation with deprecation msgs using `make html-noplot`. The documentations are build but there is no deprecation msgs. However, I have added indents. Please confirm. Done